src/FormField/MultiLine.php

```diff
@@ -517,13 +546,60 @@ public function setModel(\atk4\data\Model $model, $fields = [], $modelRef = null
                 'isEditable'  => $field->isEditable(),
                 'isHidden'    => $field->isHidden(),
                 'isVisible'   => $field->isVisible(),
-                'width'       => $width,
+                'values'      => $enumValues,
```

Comment:
If I get it correctly, for a DropDown in 'values' should be the Dropdown options. This at the moment only works for Fields with `enum` or `values` property set (or references like hasOne). If its set in Fields 'ui' property (e.g. `'ui' => ['form' => ['DropDown', 'values' => [1 => 'Yes', 2 => 'Even more Yes']]]`, its not respected.